scripts/pr.sh

```diff
@@ -75,7 +75,6 @@ if [ ! -z "$CHECK" ]; then
 fi
 
 if [ ! -z "$ALSO_TEST" ]; then
-    cargo nextest run --workspace --all-features
```

Comment:
Why remove cargo nextest ?

Reply:
Nextest It needs to be installed separately, and I want to modify the script later and add it back.